cypress/support/elements/sage-modeler-simulation-settings-elements.js

```diff
@@ -0,0 +1,19 @@
+const SageModelerSimulationSettingsElements = {
+
+    BTN_SIMULATION_SETTINGS: 'div.tool-panel div.tool-button.icon-codap-options',
+    PANEL_SIMULATION_SETTINGS: '.inspector-panel .inspector-panel-content .simulation-panel .simulation-radio-buttons',
+    RADIO_MODEL_DIAGRAM: '.simulation-panel .simulation-radio-buttons input[type=radio][value=0]',
+    RADIO_STATIC_EQUILIBRIUM_SIMULATION: '.simulation-panel .simulation-radio-buttons input[type=radio][value=1]',
+    RADIO_DYNAMIC_TIMEBASED_SIMULATION: '.simulation-panel .simulation-radio-buttons input[type=radio][value=2]',
+
+    PANEL_RELATIONSHIPS_SETTINGS: '.inspector-panel .inspector-panel-content .simulation-panel .complexity-radio-buttons',
+    RADIO_BASIC_RELATIONSHIPS: '.simulation-panel .complexity-radio-buttons input[type=radio][value=0]',
+    RADIO_EXPANDED_SET_OF_RELATIONSHIPS: '.simulation-panel .complexity-radio-buttons input[type=radio][value=1]',
```

Comment:
Same thing here, may be better to find the radio buttons by their text rather than their value.